src/compiler/scala/tools/nsc/backend/jvm/opt/BytecodeUtils.scala

```diff
@@ -119,7 +120,12 @@ object BytecodeUtils {
 
   def isNativeMethod(methodNode: MethodNode): Boolean = (methodNode.access & ACC_NATIVE) != 0
 
-  def hasCallerSensitiveAnnotation(methodNode: MethodNode): Boolean = methodNode.visibleAnnotations != null && methodNode.visibleAnnotations.asScala.exists(_.desc == "Lsun/reflect/CallerSensitive;")
+  // cross-jdk
+  def hasCallerSensitiveAnnotation(methodNode: MethodNode): Boolean =
+    methodNode.visibleAnnotations != null &&
+    methodNode.visibleAnnotations.stream.filter(ann =>
+      ann.desc == "Lsun/reflect/CallerSensitive;" || ann.desc == "Ljdk/internal/reflect/CallerSensitive;"
```

Comment:
Would this fix the issue I described in the comment below?

https://github.com/scala/bug/issues/10817#issuecomment-385190181

If so, it would be nice to backport it to Scala 2.12.

Reply:
Very clever to notice. But I thought only Spark was still stuck on 2.12. I'll try to make time to sort that out, next a modern version of Java is on my class path.

Reply:
Libraries typically support multiple Scala versions. Kafka supports Scala 2.11 and 2.12 today and it will support 2.13 soonish.

Reply:
Sorry, that was my Spark joke for the day. Some people use sbt using 2.10. But thanks again, it's fun to have an opportunity to explore the Java 9/10/11 event horizon.

Reply:
:)

Reply:
@som-snytt https://github.com/scala/scala/pull/7090